en/firefox/features/password-manager-2023.ftl

```diff
@@ -14,7 +14,12 @@ password-manager-firefox-securely-stores-your = { -brand-name-firefox } securely
 
 # Variables:
 #   $fxa (url) = link to https://www.mozilla.org/firefox/accounts/
-password-manager-with-a-free-firefox-account = With a <a href="{ $fxa }">free { -brand-name-firefox-account }</a> you can securely sync your passwords across all your devices.
+password-manager-with-a-free-mozilla-account = With a <a href="{ $fxa }">free { -brand-name-mozilla-account }</a> you can securely sync your passwords across all your devices.
+
+# Obsolete string
+# Variables:
+#   $fxa (url) = link to https://www.mozilla.org/firefox/accounts/
+password-manager-with-a-free-firefox-account = With a <a href="{ $fxa }">free { -brand-name-firefox } account</a> you can securely sync your passwords across all your devices.
```

Comment:
We should leave this unchanged as it is to an obsolete string. No need to make it a new string to be translated.

Reply:
Insert extra line space to separate the obsolete string from the string below

Reply:
Sorry, I don't quite understand the first comment?